apps/storybook/.storybook/main.ts

```diff
@@ -5,20 +5,22 @@ const config: StorybookConfig = {
   stories: ['../src/**/*.mdx', '../src/**/*.stories.tsx'],
   framework: '@storybook/react-vite',
   addons: [
+    {
+      name: '@storybook/addon-essentials',
+      options: { docs: false }, // `addon-docs` needs to be configured separately
+    },
     {
       name: '@storybook/addon-docs',
       options: {
         mdxPluginOptions: {
           mdxCompileOptions: {
-            remarkPlugins: [remarkGfm],
+            remarkPlugins: [remarkGfm], // https://storybook.js.org/docs/writing-docs/mdx#markdown-tables-arent-rendering-correctly
           },
         },
       },
     },
     '@storybook/addon-links',
-    '@storybook/addon-essentials',
   ],
-  docs: { autodocs: true },
```

Comment:
Replaced with `tags: ['autodocs']` in `.preview.ts` as per Storybook's documentation. Not sure in which version this was changed.